src/Components/Music/music.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import s from './music.module.css'
+
+const Music = () => {
+    return (
+        <div className={s.main}>
+            Music've been here
+    </div>
```

Comment:
I think there's a problem with indentation - don't forget to press "Shift + Alt + F".

Reply:
i added eslint config and prettier. And now i can use this combination